script/config-center/zk/zk-config.sh

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env bash
+# ----------------------------------------------------------------------------
+#  Copyright 2001-2006 The Apache Software Foundation.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+# ----------------------------------------------------------------------------
+#
+#   Copyright (c) 2001-2006 The Apache Software Foundation.  All rights
+#   reserved.
+
+
+# The purpose is to sync the local configuration(config.txt) to zk.
+# This script need to rely on zk.
+
+#if [[ -z "$1" ]]; then
+#	echo "zk address is empty, Please enter zk address!"
+#	exit 1
+#fi
+#
+#if [[ -z "$2" ]]; then
+#	echo "zkHome is empty, please enter zk home!"
+#	exit 1
+#fi
+
+for line in $(cat zk-params.txt); do
+	key=${line%%=*}
+	value=${line#*=}
+	case ${key} in
+	zkAddr)
+		zkAddr=${value}
+		;;
+	zkHome)
+		zkHome=${value}
+		;;
+	*)
+		echo "invalid param"
+		exit -1
+		;;
+	esac
+done
+
+if [[ -z ${zkAddr} || -z ${zkHome} ]]; then
+	echo "Incomplete parameters, please fill in the complete parameters: zkAddr:$zkAddr, zkHome:$zkHome"
+	exit -1
+fi
+
+root="/seata"
+tempLog=$(mktemp -t zk-config.log)
+
+echo "zk address is $zkAddr"
+echo "zk home is $zkHome"
+echo "zk config root node is $root"
+
+function check_node() {
+	$2/bin/zkCli.sh -server $1 ls ${root} >/dev/null 2>${tempLog}
```

Comment:
Looks like the `zkCli.sh` not present in project.

Reply:
> Looks like the `zkCli.sh` not present in project.

This needs to depend on the zk project